### PR TITLE
Update UI Documentation

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
Update UI documentation to include information on release process for @aqueducthq/common and up to date info on development workflow

## Related issue number (if any)
N/A

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/README.md [hunk 1]
```diff
@@ -60,10 +67,38 @@ To enable local development with automatic reloading, open up two terminal windo
...
+
+At the moment, we aren't following semantic versioning, but rather bumping the version number on each release of the
+aqueduct client. 
+
+When developing, please use the following pattern:
+```v0.0.1-rc0```, ```v0.0.1-rc1```, etc. for changes in active devlopment that need to be shared with the enterprise version.
+
+The suffix ```-rc0``` signifies that the current version is a release candidate for version 0.0.1. 
+
+When finished developing, update the version number to ```0.0.1``` and release alongside the Aqueduct client.
```
Comment (cw75) [inline src/ui/README.md:95]: "alongside other Aqueduct components"

**Review by cw75 — APPROVED** [review]
Left a nit in the doc otherwise LGTM!

File: src/ui/README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
 # Aqueduct UI
 
-The Aqueduct UI is implemented in Next.js. Once you've installed the Aqueduct
-package, you can start the UI by running `aqueduct start`.
+The Aqueduct UI is implemented in React and Typescript. Once you've installed the Aqueduct
```
Comment (vsreekanti) [inline src/ui/README.md:3]: Let's make it clear that installing the Aqueduct package is done via `pip` and not `npm` or something else.

**Review by vsreekanti — APPROVED** [review]

File: src/ui/README.md (hunk 2 above)
Comment (agiron123) [reply src/ui/README.md:4]: Sounds good, adding that in now.

**Pull request merged**